### ISSUE TITLE
Charts use images which do not support arm64 architecture

### ISSUE THREAD

**Issue by samip5** [bitnami/charts#7040]
**Which chart**:
postgresql, 10.4.2
redis, 14.1.1

**Describe the bug**
Try to use redis or postgresql charts on arm64, and the result is `standard_init_linux.go:228: exec user process caused: exec format error`

**To Reproduce**

1. Install kubernetes on an arm64 host
2. Try to use charts
3. Fail

**Expected behavior**

**Version of Helm and Kubernetes**: 

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.6.2", GitCommit:"ee407bdf364942bcb8e8c665f82e15aa28009b71", GitTreeState:"clean", GoVersion:"go1.16.5"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.2", GitCommit:"092fbfbf53427de67cac1e9fa54aaa09a28371d7", GitTreeState:"clean", BuildDate:"2021-06-16T12:59:11Z", GoVersion:"go1.16.5", Compiler:"gc", Platform:"linux/arm64"}
Server Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.3", GitCommit:"ca643a4d1f7bfe34773c74f79527be4afd95bf39", GitTreeState:"clean", BuildDate:"2021-07-15T20:59:07Z", GoVersion:"go1.16.6", Compiler:"gc", Platform:"linux/arm64"}
```

**Additional context**

Authentik uses your Charts as dependencies.
Reference: https://github.com/goauthentik/helm/issues/16

**Comment by rafariossaa** [comment]
Hi are you using the default images in the charts ? There are not ARM64 images by Bitnami right now but it is in our road map.

**Comment by samip5** [comment]
> Hi are you using the default images in the charts ? There are not ARM64 images by Bitnami right now but it is in our road map.

Yes, I am which is the issue.

**Comment by rafariossaa** [comment]
The issue is that the charts uses [this image](https://github.com/bitnami/charts/blob/aab51cd17357bf8f4e90803af5d190b4a6c72a4c/bitnami/postgresql/values.yaml#L61), and it [has not ARM64 architecture.](https://hub.docker.com/r/bitnami/postgresql/tags?page=1&ordering=last_updated), so it is trying to use AMD64 images in your deployment.

**Comment by samip5** [comment]
> The issue is that the charts uses [this image](https://github.com/bitnami/charts/blob/aab51cd17357bf8f4e90803af5d190b4a6c72a4c/bitnami/postgresql/values.yaml#L61), and it [has not ARM64 architecture.](https://hub.docker.com/r/bitnami/postgresql/tags?page=1&ordering=last_updated), so it is trying to use AMD64 images in your deployment.

Yes, I am aware thus the issue as it should be tracked here probably.
I had to specify the official postgres image for it to work, due to the architecture thing. 

Bitnami is probably one of the few big ish corps that have many images,  but which are not multi arch.  Most of the images from big publishers have and64 and arm64 at least supported.

**Comment by rafariossaa** [comment]
Yes, we know. 
They are in our road map, we hope to have this soon, but not ETA yet.

**Comment by samip5** [comment]
I think it should be kept open?

**Comment by juan131** [comment]
Yep! Reopening it and adding the `on-hold` tag to avoid the issue to be closed again.

**Comment by carrodher** [comment]
Hi all, we are more than happy to announce that from now on **the Bitnami container catalog is available as multi-arch in [Docker Hub](https://hub.docker.com/u/bitnami)** 🎉 🎉 

<img width="1293" alt="Screenshot 2023-02-24 at 08 16 50" src="https://user-images.githubusercontent.com/13216600/221133041-4499b10c-e57e-4756-b20c-69867a97fb5c.png">

This means you don’t need to specify anything when pulling the container images from [Docker Hub](https://hub.docker.com/u/bitnami), Docker (or any other software) will automatically pull the container image matching the host platform from which the pull command was issued.

[Here](https://blog.bitnami.com/2023/02/bitnami-arm-containers-available-at.html) you can find more info about this announcement.

-----

As usual, please create a new GitHub issue if you would like to report any bug or problem or directly contribute by creating a PR. [Here](https://github.com/bitnami/containers/blob/main/CONTRIBUTING.md) you can find the contributing guidelines.
